app/css/styles.css

```diff
@@ -0,0 +1,112 @@
+/*nav*/
+.trolley {
+    height: 50px;
+    margin: 5px;
+}
+
+/*book list*/
+h1 {
+    margin: 0;
+    padding: 20px 0;
+}
+
+.title {
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
This is cutting off the text - this requires an additional line height of 20px.

Reply:
approved